projects/VitalRecord/IJE.cs

```diff
@@ -550,5 +550,26 @@ protected void IJEToPresence(string value, Func<bool, bool> field, Func<bool, bo
                 field(false);
             }
         }
+
+        /// <summary>Checks if the given race exists in the record for Mother.</summary>
+        protected string Get_MotherRace(string name)
```

Comment:
These could move to a new IJENatality.cs file in BFDR. Then IJEBirth and IJEFetalDeath can inherit from IJENatality.